src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2302,6 +2302,20 @@ def aks_nodepool_snapshot_create(cmd,    # pylint: disable=too-many-locals,too-m
     return client.create_or_update(resource_group_name, snapshot_name, snapshot, headers=headers)
 
 
+def aks_nodepool_snapshot_update(cmd, client, resource_group_name, name, tags):   # pylint: disable=unused-argument
```

Comment:
```suggestion
def aks_nodepool_snapshot_update(cmd, client, resource_group_name, snapshot_name, tags):   # pylint: disable=unused-argument
```
should match with the declaration in `_params.py` otherwise the hep message will prompt incorrect information

![image](https://github.com/Azure/azure-cli-extensions/assets/81607949/143e70c6-c235-4984-90b7-3fe5c2e6875f)

Reply:
fixed